scripts/asm-installer/install_asm

```diff
@@ -861,6 +875,10 @@ parse_args() {
         ENABLE_REGISTRATION=1
         shift 1
         ;;
+      --enable_namespace_creation | --enable-namespace-creation)
```

Comment:
We need to also add this to the two usage prompts.

Reply:
Done